src/llmcompressor/pipelines/sequential/helpers.py

```diff
@@ -112,16 +128,20 @@ def trace_subgraphs(
     graph.device = model.device
 
     # perform subgraph partition
-    partitions = topological_partition(graph, sequential_targets)
+    partitions = topological_partition(graph, targets)
     subgraphs = partition_graph(model, partitions)
     trace_consumed_names(subgraphs)
 
+    if len(subgraphs) != len(targets) + 1:
```

Comment:
why the `+ 1` here?

Reply:
As currently implemented, the `topological_partition` function generates an extra subgraph at the beginning which does not contain a target. This adds a little more runtime, and could be removed in a separate PR

Reply:
I've added a note clarifying this in the code